lib/sumsub/webhook_sender.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Sumsub
+  # https://developers.sumsub.com/api-reference/#verifying-webhook-sender
+  class WebhookSender
+    def self.get_payload(params)
```

Comment:
Do we need this prettify thing? Not sure, but `params.to_json` doesn't suffice our needs here?

Reply:
> Do we need this prettify thing?
Yes, adding this feature took a lot of my time.

Reply:
> Not sure, but params.to_json doesn't suffice our needs here?

No.

Reply:
Do you mind in elaborate a bit more? I have no idea how to offer support in this code in case it's needed. Why does it work with this "prettified approach", and why it doesn't work with `to_json`? I think some comments explaining all the logic would be helpful in the code.

I know this is a tricky part of their API. When I was using sumsub in a project I quite from trying to make this webhook validation work (I created my own logic to ensure security). Not even sumsub support could give me a simple example with this working :see_no_evil:

Reply:
To test the webhook, support advised to use the following website: https://webhook.site/

Steps to
> Get a payload as it is without any alteration or converting to json
from https://developers.sumsub.com/api-reference/#verifying-webhook-sender

1. Set up a webhook. https://prnt.sc/26c6ixv
2. Click on the "Test" button
3. Follow the link to the webhook and click on the "Copy". https://prnt.sc/26c6iyt
4. Paste it to the rails console: https://prnt.sc/26c6hu4

Through trial and error, I got the desired result.